src/views/ManageList.jsx

```diff
@@ -1,19 +1,25 @@
 import { useState } from 'react';
-import { addItem } from '../api';
+import { addItem, shareList } from '../api';
 
-export function ManageList({ listPath }) {
+export function ManageList({ listPath, currentUserId }) {
 	const INITIAL_DATA = {
 		itemName: '',
 		daysUntilNextPurchase: '7',
 	};
 
 	const [formData, setFormData] = useState(INITIAL_DATA);
+	const [recipientEmail, setRecipientEmail] = useState('');
 
 	function handleChange(e) {
 		const { name, value } = e.target;
 		setFormData((data) => ({ ...data, [name]: value }));
 	}
 
+	function handleInviteChange(e) {
+		const { value } = e.target;
+		setRecipientEmail(value);
```

Comment:
Suggestion on making the email value case insensitive. When using a valid email address with mixed cases, the user currently receives an alert that says "The user you are trying to invite does not exist."

Reply:
I'm going to do some tests now. Valid emails in this case are the ones that are added to the users collection in firebase. I am going to test by adding an email with uppercase and see if the current setup will work. My assumption is that it will validate whatever the user email is saved as in the firebase collection. Will follow up with results.

Reply:
I added to the firebase database a new email with an uppercase as the first character. Then added a TEST uid number. From there I logged into using my real email address and shared a list with the uppercase test email. It successfully validated the test user. As long as recipient users do have an account and sender is entering the email correctly it will valid the share.

Reply:
Did a little more digging on this - it seems Firebase does not support direct case-insensitive queries. To make the search truly case-insensitive, it seems we would have to add an additional field in the Firestore documents that stores a lowercase version of the email to query a lowercase version of the recipient email against. However, since we currently require users to sign up with a google account, the email will be recorded as entirely lowercase. I added the following helper text next to the email input field: "Please enter email using lowercase letters only."